.github/workflows/nightly.yml

```
@@ -431,6 +431,10 @@ jobs:
         with:
           node-version: "20"
 
+      - name: Setup Bun
+        if: needs.decide.outputs.should_publish != 'true' || steps.current_head_prebuild.outputs.still_current == 'true'
```

actionlint (1.7.12)

[error] property "current_head_prebuild" is not defined in object type {}

(expression)